cardano-base/README.md

```diff
@@ -0,0 +1,7 @@
+# `cardano-base`
+
+This package contains a collection of Cardano-related utilities.
+
+## Feature flags
+
+See the [Haddocks](https://intersectmbo.github.io/cardano-base/cardano-base/Cardano-Base-FeatureFlags.html).
```

Comment:
There wlll be a landing page, which is more future proof that pointing to a single module
```suggestion
See the [Haddocks](https://intersectmbo.github.io/cardano-base/cardano-base).
```